db/migrate/20220918165443_add_start_date_due_date_and_overdue_to_notification_settings.rb

```diff
@@ -0,0 +1,9 @@
+class AddStartDateDueDateAndOverdueToNotificationSettings < ActiveRecord::Migration[7.0]
+  def change
+    change_table :notification_settings, bulk: true do |t|
+      t.column :start_date, :integer, default: 24
```

Comment:
From the ticket:
> - if this is done via defaults on the database layer, default should be set for all properties of the table and the value setting in Users::SetAttributesServices should be replaced by that.

If I understand this correctly, if we choose to set the defaults on the database level (which is the case), then we should set all the other notification settings defaults on the database level as well. Furthermore we should remove the existing default values from the [`Users::SetAttributesServices#initialize_notification_settings`](https://github.com/opf/openproject/blob/caa92bf39531e1098ab5592f3c4bebecee9b7c0b/app/services/users/set_attributes_service.rb#L64-L66).

@ulferts Please let us know if my understanding is not correct.

Reply:
We can leave these settings as is for now, and move the defaults of the other fields in [this task](https://community.openproject.org/projects/openproject/work_packages/43676)